app/models/miq_queue.rb

```diff
@@ -101,10 +101,7 @@ def self.put(options)
     options = options.reverse_merge(
       :priority     => NORMAL_PRIORITY,
       :queue_name   => "generic",
-      :role         => nil,
-      :server_guid  => nil,
       :msg_timeout  => TIMEOUT,
-      :deliver_on   => nil
```

Comment:
And to be honest, this is defaulting `deliver_on` to a nil (which ruby already does)